dapps/v2Explorer/src/components/QRCode.tsx

```diff
@@ -2,15 +2,15 @@ import React, {useMemo} from 'react';
 import {Svg} from 'react-native-svg';
 import {View, Image, StyleSheet} from 'react-native';
 import {QrCodeUtil} from '../utils/QrCode';
-import WCLogo from '../assets/WCGradientLogo.png';
+import WCIsotype from '../assets/WCIsotype.png';
 
 interface Props {
   uri: string;
   size: number;
   theme?: 'light' | 'dark';
```

Comment:
Do these strings need to be tied to a const?

Reply:
i could create a type with those constants, following RN core code

<img width="594" alt="Screen Shot 2023-03-27 at 12 40 32" src="https://user-images.githubusercontent.com/25931366/227992083-1bdc83f6-7007-4cf9-8974-4c95be0303a2.png">